### DISCUSSION TITLE
i am new open source project.

### DISCUSSION THREAD

**Discussion by darkb0ts** [mitmproxy/mitmproxy#6765] [Q&A]
i am new open source contribution  and where need to start and i search any good first issue. i got nothing

**Comment by rosydawn6** [comment]
Welcome!  Here are a few suggestions.  Work through this: https://github.com/mitmproxy/mitmproxy/blob/main/CONTRIBUTING.md

Then look for the specific issue referenced there: https://github.com/mitmproxy/mitmproxy/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
